cmake/features.cmake

```diff
@@ -15,6 +15,8 @@ endif()
 
 ie_option (ENABLE_TESTS "unit, behavior and functional tests" OFF)
 
+ie_option (ENABLE_SKIPPED_TESTS "run skipped tests" OFF)
```

Comment:
What is for cases with `DISABLE_` prefix?
Do we have a plan how to fix such tests? Or single layer tests don't contain tests with prefix `DISABLE_` and we have a guideline that we shouldn't use this prefix.

Reply:
Second option, our recommendation was to use `[tests_skip_config](https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp)` instead of `DISABLE_` prefix